package.json

```diff
@@ -21,17 +21,23 @@
     "typings"
   ],
   "scripts": {
-    "test": "nyc --extension --reporter=lcov --reporter=text-summary vows -- --spec test/*-test.js"
+    "build": "tsc",
+    "test": "tsc && nyc --extension --reporter=lcov --reporter=text-summary vows -- --spec test/*-test.js"
```

Comment:
would prefer to keep the test script separate from the build script. in case we're ever only working on tests, there'd be no need to recompile the main library